app/models/mixins/vmdb_database_metrics_mixin.rb

```diff
@@ -9,7 +9,7 @@ def first_or_last_capture(interval_name, sort_order)
     my_metrics
       .where(:capture_interval_name => interval_name)
       .select(:timestamp)
-      .order("timestamp #{sort_order}")
+      .order(sort_order == "DESC" ? "timestamp DESC" : "timestamp ASC")
```

Comment:
I mean... the sanitization above avoids said SQL injection.

I guess as either option would work anyway as it's always sorted... ok.